micropython/micropython-1.15-patchset/py.patch

```diff
@@ -86,8 +92,17 @@ diff -Naur micropython-1.15/py/modmath.c micropython-phoenix/py/modmath.c
  #if MICROPY_PY_BUILTINS_FLOAT && MICROPY_PY_MATH
  
 diff -Naur micropython-1.15/py/mpconfig.h micropython-phoenix/py/mpconfig.h
---- micropython-1.15/py/mpconfig.h	2021-04-14 07:35:31.000000000 +0200
-+++ micropython-phoenix/py/mpconfig.h	2021-04-14 13:27:45.000000000 +0200
+--- micropython-1.15/py/mpconfig.h	2021-04-18 16:27:07.000000000 +0200
++++ micropython-phoenix/py/mpconfig.h	2021-05-25 09:43:17.359538140 +0200
+@@ -805,7 +805,7 @@
+ 
+ // Whether to use internally defined *printf() functions (otherwise external ones)
+ #ifndef MICROPY_USE_INTERNAL_PRINTF
+-#define MICROPY_USE_INTERNAL_PRINTF (1)
++#define MICROPY_USE_INTERNAL_PRINTF (0)
```

Comment:
should be provided by define in `CFLAGS` instead of patching the default value

Reply:
As it is part of `mpconfigport.mk` that is substituted, not patched, this review is outdated